src/sparse/impl/KokkosSparse_spmv_bsrmatrix_impl.hpp

```diff
@@ -555,24 +559,25 @@ struct BSR_GEMV_Functor {
   typedef typename AMatrix::non_const_size_type size_type;
 
   const value_type alpha;
+  const value_type beta;
+
   AMatrix m_A;
   XVector m_x;
   YVector m_y;
 
   const ordinal_type block_dim;
```

Comment:
Why not just declare this member as `const int` that way you would not need to cast it to int everywhere?

Reply:
Done.